### DISCUSSION TITLE
JvDBGrid Issue

### DISCUSSION THREAD

**Discussion by Karimmagdy111** [project-jedi/jedi#16] [General]
JvDBGrid has much flicker on form VCL Delphi, Is there a solution for that

**Comment by ronaldhoek** [comment]
It depends on what you are doing with the grid (like custom drawing, data changing) in your application because I don't see this in our application (which uses TJvDBGrid a lot)

**Reply by Karimmagdy111** [reply]
I using it for preview, insert, update and delete data with default options

**Reply by Karimmagdy111** [reply]
I test it again by adding JvDBGrid without datasource and add some fileds then run code
when moving mouse on grid, columns titles filcker
(there is no events on grid)

**Reply by Karimmagdy111** [reply]
https://user-images.githubusercontent.com/39433467/225007459-9a9936f8-3482-4312-b926-65009f239a83.mp4

**Reply by ronaldhoek** [reply]
Hhhm, strange - I'm unable to reproduce the issue (using Delphi XE2/10.3)
- maybe a Delphi version-specific issue?

**Reply by ronaldhoek** [reply]
https://user-images.githubusercontent.com/5037087/225026062-facf688e-708e-4dac-b9bf-1d1a8ee95559.mp4

**Reply by Karimmagdy111** [reply]
I'm using Delphi 10.4

**Reply by Karimmagdy111** [reply]
Thanks, I Found It
It appears when using bdRightToLeft

**Reply by acgubamg** [reply]
cell is buttons?
